article/_posts/2019-07-25-SphericalCNNCortexParcellation.md

```diff
@@ -0,0 +1,115 @@
+as the number of atlases increases. Unfortunately, inter-subject registration
+is unavoidable in this approach to align multiple atlases.
+
+Traditional CNNs architectures are still immature in handling non-uniform data
+with high complexity due to the Euclidean space coherence incorporated with
+existing deep architectures.
+
+Spherical CNNs have recently emerged to deal with spherical domain data. Authors
+propose a novel cortical parcellation approach using a deep spherical U-Net
+encoding surface mesh features.
+
+
+# Methods
+
+To compute the input features to the CNN, the following features are computed:
+- The **deformation field**: they use a spherical surface registration method
+that reconstructs the deformation field by a linear combination of spherical
+harmonics coefficients with degrees $$l=0 \cdot 10$$.
+- The **boundary map**: authors compute deformation fields that align parcel
+boundaries for more accurate prediction.
+
+The **deformed data** features used to feed to the network are then:
+- The **mean curvature** ($$$iH$) from inflated surface.
+- The **sulcal depth** ($$SD$$).
+- The **mean curvature from cortical surface** ($$H$$).
+
+To create a template, they co-register training samples in an iterative manner
+and compute a distance map of the mode (most frequent) cortical labels across
+the training set after their registration to the template using the three
+geometric features. They then register the normalized distance map to the
+template distance map to produce the deformation fields.
+
+![](/article/images/SphericalCNNCortexParcellation/Architecture.png)
+
+Authors used a spherical U-net architecture designed for segmentation tasks.
+They provide the described geometric features to the input channels and the
+corresponding labels to the output channels.
+
+Data augmentation is performed by using all the deformations of the spherical
+harmonics between $$l=0 \cdot 10$$.
+
+At the end of the testing stage, they **refine predicted parcellation maps with
+a standard graph-cut** method to remove potential isolated regions and to create
+smooth parcel boundaries.
+
+## Data
+
+427 T1-weighted 3T MRI images.
+
+
+# Results
+
+The cortical surfaces and their spherical mapping were reconstructed via a
+standard FreeSurfer pipeline.
+
+The baselines used were:
+- A spherical U-Net model driven only by non-rigid deformation information.
+- A multi-atlas and spherical U-Net model with the rigid deformation
+information.
+
+For a fair comparison, the same graph-cut technique was applied to all the
+baseline methods.
+
+The authors' approach outperforms multi-atlas (46 regions) and spherical U-Net
+(24 regions).
+
+No regions were found with significantly reduced Dice overlap.
+
+![](/article/images/SphericalCNNCortexParcellation/Results_images.png)
+![](/article/images/SphericalCNNCortexParcellation/Results_tables.png)
+
+
+# Conclusions
+
+Authors presented a cortical parcellation method using spherical U-Net with
```

Comment:
The auhtors present a ... using a spherical U-Net ...

Reply:
Done in  584ea4d.